script/release/kie-copyBinariesToFilemgmt.sh

```diff
@@ -90,7 +90,8 @@ scp -r droolsjbpm-knowledge/kie-api/target/apidocs/* $droolsDocs/$version/kie-ap
 scp -r droolsjbpm-tools/droolsjbpm-tools-distribution/target/droolsjbpm-tools-distribution-$version/droolsjbpm-tools-distribution-$version/binaries/org.drools.updatesite/* $jbpmHtdocs/$version/updatesite
 scp jbpm/jbpm-distribution/target/jbpm-$version-bin.zip $jbpmHtdocs/$version
 scp jbpm/jbpm-distribution/target/jbpm-$version-examples.zip $jbpmHtdocs/$version
-scp kie-wb-distributions/kie-server-distributions/jbpm-server-distribution/target/jbpm-server-$version-dist.zip $jbpmHtdocs/$version
+scp kie-wb-distributions/business-central-parent/business-central-distribution-wars/business-central/target/business-central-$version-*.war $droolsHtdocs/$version
```

Comment:
this line should move up to # copies drools binaries to filemgmt.jboss.org section as the eap7 and wildfly14 war will be uploaded to drools@filemgmt.jboss.org
The old scp line of wars was removed in here https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/d38e2c137aee9bfe66a514e5fddbfb587ea4661e#diff-27f79d511912976f68b73ed9bb022b3fL83

Reply:
Just updated the commit, would you please take a look again? Thanks!